sdk/Eversign/Client.php

```diff
@@ -434,6 +434,20 @@ public function downloadFinalDocumentToPath(Document $document, $path, $auditTra
         }
         return $this->downloadDocumentToPath($document, $path, $auditTrail);
      }
+    
+    /**
+     * Downloads the Audit Trail to the specified Path.
+     * Returns true if saving was successful, otherwise false.
+     * @param \Eversign\Document $document
+     * @param string $path
+     * return boolean success
+     */
+     public function downloadAuditTrail(Document $document, $path, $auditTrail = true, $documentId = "AT"){
```

Comment:
Since $documentId will only be used with "AT" - Audit Trail, we can remove $documentId = "AT" from here, end result
 public function downloadAuditTrail(Document $document, $path, $auditTrail = true) {
and assign it directly in return $this->downloadDocumentToPath($document, $path, $auditTrail, "AT");